.changeset/healthy-mayflies-end.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/experimental-app-router': minor
```

Comment:
I don't think this is breaking since it fixes a security recommendation which does not change the usage or the way that users should use the package.

Reply:
@blakewilson Thoughts?

Reply:
@matthewguywright @theodesp  I think that makes sense. The refresh token cookie we use is not part of our "public api" and so modifying it should not come as a surprise to anyone.

Reply:
@matthewguywright Since we agreed it's not a breaking change, let's update the changeset type to `patch`